cilium/CFP-27752-operator-manages-cilium-identities.md

```diff
@@ -0,0 +1,225 @@
+# CFP-27752: Operator Manages Cilium Identities
+
+**Authors:** dordel@google.com
+
+**Area:** SIG: SIG-POLICY, SIG-AGENT, SIG-K8S
+
+**Target Cilium Release**: 1.17
```

Comment:
```suggestion
**Target Cilium Release**: 1.18
```

Reply:
(Though this may still be optimistic with the kvstore compatibility)

Reply:
Applied change